<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU模拟</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_Open">
      <property name="text">
       <string>开始监视</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Close">
      <property name="text">
       <string>取消监视</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CRT_CPUView" name="widget_1" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="CRT_CPUView" name="widget_2" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="CRT_CPUView" name="widget_3" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="CRT_CPUView" name="widget_4" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CRT_CPUView</class>
   <extends>QWidget</extends>
   <header location="global">crt_cpuview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
